modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/store/ObjectStoreSettingsConverter.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.store;
+
+
+import static org.mule.runtime.api.util.Preconditions.checkArgument;
+
+import org.mule.runtime.api.store.ObjectStoreSettings;
+
+/**
+ * Converts Object Store settings between the SDK api {@link org.mule.sdk.api.store.ObjectStoreSettings} and the Mule opi
+ * {@link ObjectStoreSettings}
+ *
+ * @since 4.5.0
+ */
+public class ObjectStoreSettingsConverter {
```

Comment:
feels like this class and `ObjectStoreExceptionConverter` should be collapsed into some kind of `SdkObjectStoreUtils` or something like that.